hyperspy/tests/io/test_sur.py

```diff
@@ -0,0 +1,826 @@
+         'H58_T_Axis_Name',
+         'H59_T_Step_Unit',
+         'H60_Comment',
+         ]
+
+def test_load_spectrum():
+    fname = os.path.join(MY_PATH, "sur_data",
+                         "test_spectrum.pro")
+    s = load(fname)
+    assert s.data.shape == (1,512)
+
+    md = s.metadata
+    assert md.Signal.quantity == 'CL Intensity a.u.'
+    assert_allclose(s.axes_manager[0].scale,1.0)
+    assert_allclose(s.axes_manager[0].offset,0.0)
+    assert_allclose(s.axes_manager[1].scale,1.084000246009964e-6)
+    assert_allclose(s.axes_manager[1].offset,172.84281784668565e-6)
+
+    assert s.axes_manager[0].name == 'T'
+    assert s.axes_manager[0].units == ''
+    assert s.axes_manager[1].name == 'Wavelength'
+    assert s.axes_manager[1].units == 'mm'
+    assert s.axes_manager[0].size == 1
+    assert s.axes_manager[0].navigate == True
+    assert s.axes_manager[1].size == 512
+    assert s.axes_manager[1].navigate == False
+
+    omd = s.original_metadata
+    assert list(omd.as_dictionary().keys()) == ['Object_0_Channel_0']
+    assert list(omd.Object_0_Channel_0.as_dictionary().keys()) == ['Header']
+    assert list(omd.Object_0_Channel_0.Header.as_dictionary().keys()) == \
+        ['H01_Signature',
+         'H02_Format',
+         'H03_Number_of_Objects',
+         'H04_Version',
+         'H05_Object_Type',
+         'H06_Object_Name',
+         'H07_Operator_Name',
+         'H08_P_Size',
+         'H09_Acquisition_Type',
+         'H10_Range_Type',
+         'H11_Special_Points',
+         'H12_Absolute',
+         'H13_Gauge_Resolution',
+         'H14_W_Size',
+         'H15_Size_of_Points',
+         'H16_Zmin',
+         'H17_Zmax',
+         'H18_Number_of_Points',
+         'H19_Number_of_Lines',
+         'H20_Total_Nb_of_Pts',
+         'H21_X_Spacing',
+         'H22_Y_Spacing',
+         'H23_Z_Spacing',
+         'H24_Name_of_X_Axis',
+         'H25_Name_of_Y_Axis',
+         'H26_Name_of_Z_Axis',
+         'H27_X_Step_Unit',
+         'H28_Y_Step_Unit',
+         'H29_Z_Step_Unit',
+         'H30_X_Length_Unit',
+         'H31_Y_Length_Unit',
+         'H32_Z_Length_Unit',
+         'H33_X_Unit_Ratio',
+         'H34_Y_Unit_Ratio',
+         'H35_Z_Unit_Ratio',
+         'H36_Imprint',
+         'H37_Inverted',
+         'H38_Levelled',
+         'H39_Obsolete',
+         'H40_Seconds',
+         'H41_Minutes',
+         'H42_Hours',
+         'H43_Day',
+         'H44_Month',
+         'H45_Year',
+         'H46_Day_of_week',
+         'H47_Measurement_duration',
+         'H48_Compressed_data_size',
+         'H49_Obsolete',
+         'H50_Comment_size',
+         'H51_Private_size',
+         'H52_Client_zone',
+         'H53_X_Offset',
+         'H54_Y_Offset',
+         'H55_Z_Offset',
+         'H56_T_Spacing',
+         'H57_T_Offset',
+         'H58_T_Axis_Name',
+         'H59_T_Step_Unit',
+         'H60_Comment',
+         ]
+
+def test_load_surface():
+    fname = os.path.join(MY_PATH, "sur_data",
```

Comment:
Just to be sure, this is CL intensity map? If so, can you confirm that there is not metadata specifying the energy range? This information would be useful to be parsed.

Reply:
It actually is not. The surface type is actually mostly used for AFM data. The only data that I could share without concerns are my CL surfaces summed along energy axis, in which case I can confirm you that there is no metadata saving the energy range.